signups/admin.py

```diff
@@ -23,3 +26,17 @@ class SignupTargetAdmin(admin.ModelAdmin):
     fields = ('name', 'identifier')
     prepopulated_fields = {'identifier': ('name',)}
     inlines = (SignupInline,)
+    actions = ["export_participants"]
+
+    def export_participants(self, request, queryset):
+        if queryset.count() > 1:
+            return self.message_user(request, _(u"Valitse vain yksi Signup Target"), messages.ERROR)
+
+        signup_target = queryset.first()
+        participants = signup_target.users.all()
+
+        response = HttpResponse(content_type='text/csv')
+        response['Content-Disposition'] = 'attachment; filename=orders.csv'
```

Comment:
Not sure why the field is named `orders` :) Also, maybe include signup target name and some kind of rough timestamp to the filename as well?